pywbemtools/pywbemcli/pywbemcli.py

```diff
@@ -554,14 +552,20 @@ def create_server_instance(connection_name):
 
         # If --name option, get server from connection
         if connection_name:
-            validate_connections_file(connections_repo)
+            # validate_connections_file(connections_repo)
             try:
                 pywbem_server = connections_repo[connection_name]
             except KeyError:
                 raise click.ClickException(
-                    "Connection definition '{}' not found in "
-                    "connections file '{}'".
+                    "Connection name '{}' not found in connections file '{}'".
                     format(connection_name, connections_repo.connections_file))
+            except ConnectionsFileError as cfe:
+                click.echo('Fatal error loading connections file: "{0}". '
+                           "Exception: {1}: {2}".
+                           format(connections_repo.connections_file,
+                                  cfe.__class__.__name__, cfe),
+                           err=True)
```

Comment:
The exception message that now comes up has everything to be displayed as is, without adding more context or exception names or connections file name. So I suggest to just display it.